server/helpers/utils.js

```diff
@@ -5,13 +5,24 @@ import bcrypt from 'bcryptjs';
  * @param {string} title
  * @return {string} unique_slug
  */
-const createUniqueSlug = title => `${slug(title, { lower: true })}-${Date.now()}`;
+export const createUniqueSlug = title => `${slug(title, { lower: true })}-${Date.now()}`;
 
 /**
  * @param {string} password
  * @return {string} hash
  */
+export const hashPassword = password => bcrypt.hashSync(password, bcrypt.genSaltSync(10));
 
-const hashPassword = password => bcrypt.hashSync(password, bcrypt.genSaltSync(10));
+/**
+ * @param {string} text
+ * @param {string} length
+ * @return {string} text
+ */
+export const ellipsis = (text, length = 200) => {
```

Comment:
I also think you should rename your function to something like createEllipses, to give a full description of what the function does.